src/components/Score.js

```diff
@@ -4,16 +4,41 @@ import { browserHistory } from 'react-router';
 import ScoreInput from './ScoreInput.js';
 
 const Score = React.createClass({
-  submitScores() {
-    browserHistory.push('/d/' + this.props.params.sessionId);
+  getInitialState() {
+    return {
+      scores: {}
+    };
+  },
+  saveScore() {
+    // TODO: validation - pk2
+    const data = JSON.stringify({
+      sessionId: this.state.sessionId,
+      scopes: this.state.scores
+    });
+    fetch('http://localhost:3000/api/score', {
```

Comment:
We may also need a `fetch` polyfill. 
<img width="1268" alt="screen shot 2016-02-19 at 9 31 45 am" src="https://cloud.githubusercontent.com/assets/3665341/13178219/a430677e-d6eb-11e5-80cf-3e437ab89aa6.png">